atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -412,7 +413,7 @@ private TransactionManager serializableInternal(@Output List<AutoCloseable> clos
 
         Refreshable<SweepConfig> sweepConfig = runtime.map(AtlasDbRuntimeConfig::sweep);
 
-        KeyValueService keyValueService = initializeCloseable(
+        KeyValueService internalKeyValueService = initializeCloseable(
```

Comment:
Driveby: I think this is the line we should hold, and eventually this will be the usage, only for internal tables.

Reply:
This is correct here, but for the future: please be careful with this kind of rename as a drive-by, reviewing this took more time than the "main" part of this PR!

I think you are doing this, but because some of the classes are used by consumers who are still directly creating their stuff differently we need to be careful about sending the wrong message. This path isn't used by the main alternative consumer so it's right, but I had to think about that

Reply:
+1 fwiw this is exactly my reasoning, it felt like one less PR.